accord-core/src/main/java/accord/utils/async/AsyncChains.java

```diff
@@ -430,8 +430,10 @@ protected Cancellable start(BiConsumer<? super V, Throwable> callback)
 
     // either the thing we start, or the thing we do in follow-up
     BiConsumer<? super V, Throwable> next;
+    private final Head<?> head;
```

Comment:
you can revert this, I did this to aid in debugger support

Reply:
its not in trunk.  The patch I sent you I needed to know the state of the `allOf` to see which shard was not replying, so doing this let me get access.  This was debugging code